index.js

```diff
@@ -10,21 +10,22 @@ function getTextValue() {
   })
   $('#cssEditorValue').append(() => {
     let cssCode = cssEditor.getValue();
-    return cssCode;
+    let cssValue = `${cssCode}`
+    return cssValue;
   })
   $('#htmlEditor').on('change', function() {
     $('#editorValue').empty();
     $('#editorValue').append(() => {
       let code = htmlEditor.getValue();
-      let cssCode = cssEditor.getValue();
       return code;
     })
   })
   $('#cssEditor').on('change', function() {
     $('#cssEditorValue').empty();
     $('#cssEditorValue').append(() => {
```

Comment:
For the sake of readability, instead of passing an anonymous function to a $.append() call, I would personally create a function that generates this string for you and set that return value to a variable outside of $.append() so you only have to pass .append() the valid HTMLstring.

Reply:
I understand that it is a stylistic preference and the approach you took here has been valid jQuery since version 1.4, but personally, I find it hard to sift through several anonymous functions worth of HTMLstring generation.  I noticed you used this exact same anonymous function several times through the code and creating a separate function for this purpose would help with code re-use.